src/main/java/com/jalasoft/sdfc/pages/contacts/ContactsPage.java

```diff
@@ -1,4 +1,42 @@
 package com.jalasoft.sdfc.pages.contacts;
 
-public class ContactsPage {
+import com.jalasoft.sdfc.core.ui.ISteps;
+import com.jalasoft.sdfc.pages.AbstractPage;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+
+public class ContactsPage extends AbstractPage {
+
+    @FindBy(css = "a[title='New']")
+    private WebElement newButton;
+
+    @FindBy(css = "input[class=\"lastName compoundBLRadius compoundBRRadius form-element__row input\"]")
+    private WebElement ContactLastNameTextField;
+
+    @FindBy(css = "button[title=\"Save\"]")
+    private WebElement saveButton;
+
+    public void clickNewButton() {
+        action.click(newButton);
+    }
+
+    public void setContactFields(Map<String, String> data) {
```

Comment:
i think this should be in another class called ContactsForm.java or something like that since there is modal form component to create contacts. I mean the modal form should be considered as an independent page object